lib/content-types/load.js

```diff
@@ -39,6 +39,17 @@ const check = (type) => {
   if (type.id !== slugify(type.id)) {
     return `${type.id} needs to be written in kebab case (e.g. ${slugify(type.id)})`;
   }
+
+  if (type.hasOwnProperty('identifier')) {
+    if (typeof type.identifier !== 'boolean') {
+      return 'Content type identifier must be a boolean';
```

Comment:
Identifier _specifically cannot_ be a boolean. `identifier` _needs_ to be a **string** and it _needs_ to be the `id` value of an attribute in the content type. I think you mistook the "remove identifier from attributes" as move it as-is to the top level of the content type instead.